base/Enums.jl

```diff
@@ -31,7 +31,7 @@ end
 """
     @enum EnumName[::BaseType] EnumValue1[=x] EnumValue2[=y]
 
-Create an [`Enum`]{`BaseType`} subtype with name `EnumName` and enum member values of
+Create an `Enum`{`BaseType`} subtype with name `EnumName` and enum member values of
```

Comment:
I would say no, but I see that the rest of the documentation says yes. :) Fixed.